src/cli-constant.js

```diff
@@ -158,6 +158,13 @@ const detailedOptions = normalizeDetailedOptions({
   "debug-print-doc": {
     type: "boolean"
   },
+  editorconfig: {
+    type: "boolean",
+    category: CATEGORY_CONFIG,
+    description: "Take .editorconfig into account when parsing configuration.",
```

Comment:
We should also add `oppositeDescription` to show docs for `--no-editorconfig`.

Reply:
Good point, that makes the `--help` message much better. Fixed in https://github.com/prettier/prettier/pull/3255/commits/64a441384dafc6dec3d51e978dc068b6bbfb585f